runtime/wasm/jslib.wat

```diff
@@ -128,6 +129,13 @@
       (ref.i31 (call $strict_equals
                   (call $unwrap (local.get 0)) (call $unwrap (local.get 1)))))
 
+   (func (export "caml_js_expr") (export "caml_pure_js_expr")
+         (export "caml_js_var") (export "caml_js_eval_string")
```

Comment:
`caml_js_eval_string` doesn't use extra parens in js but does in wasm.

Reply:
Oh, indeed! I missed that. The Wasm code was wrong.